PythonTool/WindowSettings.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from visit import *
+
+
+def WindowSettings(Shading=False):
```

Comment:
Yes. But for that to have any effect, arguments will have to added to each script that used WindowSettings up to MakeImagesTerminal.py.